module/zfs/vdev_raidz.c

```diff
@@ -196,6 +196,38 @@ vdev_raidz_map_free_vsd(zio_t *zio)
 	}
 }
 
+typedef struct reflow_node {
+	uint64_t re_txg;
+	uint64_t re_logical_width;
```

Comment:
could you write a comment explaining the exact meaning of these, e.g. `Blocks whose logical birth is >= re_txg (and < the next entry's re_txg) use re_logical_width.`

Reply:
BTW do we need to wrap avl tree in mutex? Or it could be assumed that avl_add() from raidz_reflow_complete_sync() and vdev_raidz_get_tsd() will not conflict with vdev_raidz_get_width()?
Also, it should be mutex or could be rw lock if true?